examples/chart/teleport-relay/templates/deployment.yaml

```diff
@@ -0,0 +1,226 @@
+            - "--"
+            - /usr/local/bin/teleport
+          args:
+            - start
+            {{- if .Values.insecureSkipProxyTLSVerify }}
+            - "--insecure"
+            {{- end }}
+            {{- with .Values.extraArgs }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          env:
+            {{- /* This variable is set for telemetry purposes.
+              Telemetry is opt-in and controlled at the auth level. */}}
+            - name: TELEPORT_INSTALL_METHOD_HELM_KUBE_AGENT
+              value: "true"
+            {{- if .Values.teleportClusterName }}
+            - name: KUBERNETES_TOKEN_PATH
+              value: /var/run/secrets/teleport-join-sa-token/join-sa-token
+            {{- end }}
+            {{- if .Values.tls.existingCASecretName }}
+            - name: SSL_CERT_FILE
+              value: /etc/teleport-tls-ca/ca.pem
+            {{- end }}
+            {{- with .Values.extraEnv }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          ports:
+            - containerPort: 3000
+              name: diagnostics
+              protocol: TCP
+            - containerPort: 3040
+              name: transport
+              protocol: TCP
+            - containerPort: 3041
+              name: peer
+              protocol: TCP
+            - containerPort: 3042
+              name: tunnel
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: diagnostics
+            failureThreshold: 6
+            initialDelaySeconds: 5
+            periodSeconds: 5
+            timeoutSeconds: 1
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: diagnostics
+            failureThreshold: 12
+            initialDelaySeconds: 5
+            periodSeconds: 5
+            timeoutSeconds: 1
+          {{- with .Values.preStopDelay }}
+          lifecycle:
+            {{- /* waiting during preStop ensures that load balancers have time
+              to stop sending new connections to the Terminating pod */}}
+            preStop:
+              exec:
+                command:
+                  - /usr/local/bin/teleport
+                  - wait
+                  - duration
+                  - {{ quote . }}
+          {{- end }}
+          volumeMounts:
+            - mountPath: /var/lib/teleport
+              name: data
+            - mountPath: /etc/teleport.yaml
+              name: config
+              subPath: teleport.yaml
+            - mountPath: /etc/teleport-secrets
+              name: auth-token
+            {{- if .Values.teleportClusterName }}
+            - mountPath: /var/run/secrets/teleport-join-sa-token
+              name: join-sa-token
+            {{- end }}
+            {{- if .Values.tls.existingCASecretName }}
+            - mountPath: /etc/teleport-tls-ca
+              name: teleport-tls-ca
+            {{- end }}
```

Comment:
Would you be willing to fix [this issue](https://github.com/gravitational/teleport/issues/59389) as it pertains to this chart? This should be a 2-3 line change outside of comments.